src/Components/App/App.js

```diff
@@ -47,26 +40,35 @@ class App extends Component {
   render() {
     return (
       <main className="App">
-        {this.state.singleMovie ? (
-          <>
-            <Header
-              returnHome={this.returnHome}
-              bannerImage={this.state.singleMovie.backdrop_path}
-            />
-            <SingleMovie movie={this.state.singleMovie} />
-          </>
-        ) : (
-          <>
-            <Header
-              returnHome={this.returnHome}
-              bannerImage={this.state.bannerImage}
-            />
-            <MoviesContainer
-              movies={this.state.movies}
-              selectMovie={this.selectMovie}
-            />
-          </>
-        )}
+        <Route
+          exact
+          path="/"
+          render={() => {
+            return (
+              <>
+                <Header bannerImage={this.state.bannerImage} />
+                <MoviesContainer
+                  movies={this.state.movies}
+                  selectMovie={this.selectMovie}
+                />
+              </>
+            );
+          }}
+        />
+        <Route
+          exact
+          path="/:id"
+          render={({ match }) => {
+            const currentMovieId = parseInt(match.params.id);
+            console.log('match', match);
+            return (
+              <>
```

Comment:
I don't think we need this fragment here and we can just directly return the SingleMovie component.